### ISSUE TITLE
After update to 1.1.40 getting error launching IE, with a caused by message that Safari isn't set

### ISSUE THREAD

**Issue by tbbstny** [serenity-bdd/serenity-core#539]
```
<serenity.version>1.1.40</serenity.version>
<serenity.maven.version>1.1.40</serenity.maven.version>
<serenity.cucumber.version>1.1.10</serenity.cucumber.version>

```

ERROR: net.thucydides.core.webdriver.WebDriverFacade - FAILED TO CREATE NEW WEBDRIVER_DRIVER INSTANCE class org.openqa.selenium.ie.InternetExplorerDriver: Could not instantiate class org.openqa.selenium.ie.InternetExplorerDriver
net.thucydides.core.webdriver.UnsupportedDriverException: Could not instantiate class org.openqa.selenium.ie.InternetExplorerDriver
	at net.thucydides.core.webdriver.WebDriverFactory.newWebdriverInstance(WebDriverFactory.java:128)
	at net.thucydides.core.webdriver.WebDriverFacade.newDriverInstance(WebDriverFacade.java:142)
	at net.thucydides.core.webdriver.WebDriverFacade.newProxyDriver(WebDriverFacade.java:133)
...
Caused by: java.lang.RuntimeException: Safari could not be found in the path!
Please add the directory containing ''Safari.exe'' to your PATH environment
variable, or explicitly specify a path to Safari like this:
*safari c:\blah\Safari.exe
	at org.openqa.selenium.browserlaunchers.locators.SingleBrowserLocator.findBrowserLocationOrFail(SingleBrowserLocator.java:42)
	at org.openqa.selenium.safari.SafariDriverCommandExecutor.start(SafariDriverCommandExecutor.java:100)
	at org.openqa.selenium.safari.SafariDriver.startClient(SafariDriver.java:76)

**Comment by wakaleo** [comment]
Thanks for spotting this, it should be fixed in 1.1.41.

**Comment by wakaleo** [comment]
Fixed in 1.1.41
